packages/teams-js/src/internal/communication.ts

```diff
@@ -1090,6 +1098,7 @@ function createMessageRequest(
     uuid: messageUuid,
     func: func,
     timestamp: Date.now(),
+    monotonicTimestamp: getCurrentTimestamp(),
```

Comment:
as i mentioned in the other comment, this should not take on "date.now" value

Reply:
Good catch. Just pushed the change to never mix up both 😄